src/screens/Asset/Asset.js

```diff
@@ -310,8 +238,10 @@ const AssetScreen = ({
             {isEtherspotAccount(activeAccount) && <HistoryList items={transactions} chain={chain} />}
           </>
         )}
-      </ScrollWrapper>
-    </ContainerWithHeader>
+      </Content>
+      <FloatingButtons items={buttons} />
+      {!isSendActive && <SWActivationCard />}
```

Comment:
Can you also add additional check here to show `SWActivationCard` based on if current account is Archanova?

Reply:
@poocart Done.